dc/s2n-quic-dc/src/path/secret/map.rs

```diff
@@ -782,6 +783,30 @@ impl dc::Endpoint for Map {
     fn new_path(&mut self, connection_info: &dc::ConnectionInfo) -> Option<Self::Path> {
         Some(HandshakingPath::new(connection_info, self.clone()))
     }
+
+    fn on_possible_secret_control_packet(
+        &mut self,
+        // TODO: Maybe we should confirm that the sender IP at least matches the IP for the
+        //       corresponding control secret?
+        _datagram_info: &DatagramInfo,
+        payload: &mut [u8],
+    ) -> bool {
+        let payload = s2n_codec::DecoderBufferMut::new(payload);
+        // TODO: Is 16 always right?
```

Comment:
Yes, SRTs are always 16 bytes

Reply:
this isn't a stateless reset token though, its the `crypto_tag_len`. Should I make this const public and grab it from here? https://github.com/aws/s2n-quic/blob/75e60fcb360f32f89aa81d74008b895d68874c73/dc/s2n-quic-dc/src/crypto/awslc.rs#L11

Reply:
They're the same thing in this case. The SRT goes in the place of the auth tag

Reply:
Isn't that just for `UnknownPathSecret`? `StaleKey` and `ReplayDetected` seem to accommodate different tag lengths

Reply:
ah good point. i guess it's fine to keep this here for now